src/brouwers/shop/models/config.py

```diff
@@ -1,13 +1,11 @@
 from django.db import models
-from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 
 from solo.models import SingletonModel
 
-from ..payments.constants import PaymentMethods
+__all__ = ["ShopConfiguration"]
```

Comment:
What does this do?

Reply:
if you do `from .config import *`, then it only pics up what's defined in `__all__` instead of all the declared names